### ISSUE TITLE
Screen reader users cannot type a city name or UTC offset in the time zone chooser

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/calendar#96]
Currently, screen reader users cannot narrow down the time zones by entering a city name or a UTC offset. I didn't know this could be done until I came back to #81 :-)

With a screen reader running, do the following:

1. Add anew calendar event.
2. Navigate to the "Time zone" combo box
3. Open the combo box.
4. Type a city name or UTC offset

**Expected result**: The displayed options match the string the user just typed, which makes arrowing up and down through the available options much more efficient.

**Obtained result**: Nothing happens, the only thing a user can do is to arrow up and down through the 142 available time zones, which makes selecting the desired one highly inefficient

**Comment by daniel-montalvo** [comment]
Still persists after redesign.

**Comment by jean-gui** [comment]
@daniel-montalvo Can you try again? The component for selecting a time zone has been improved and can now be searched

**Comment by daniel-montalvo** [comment]
This is working so much better now @jean-gui  I will close this, no need to keep this open.
